ecl/eclcc/eclcc.cpp

```diff
@@ -2525,8 +2531,6 @@ void EclCC::processBatchedFile(IFile & file, bool multiThreaded)
             if (info.wu &&
                 (info.wu->getDebugValueBool("generatePartialOutputOnError", false) || info.queryErrorProcessor().errCount() == 0))
             {
-                WuStatisticTarget statsTarget(info.wu, "eclcc");
-                gatherTransformStats(statsTarget);
```

Comment:
This has now been made conditional rather than unconditional, and the option has been added to the eclcc help.
@shamser what kind of documentation do you suggest?